src/util/double_print.py

```diff
@@ -0,0 +1,7 @@
+"""Utility function to print out to stdout and file at the same time."""
+
+
+def double_print(outfile, line_string):
+    """Write to console and a file using the same string."""
+    print(line_string)
+    outfile.write(line_string + "\n")
```

Comment:
Prefer Python standard library logging to this kind of thing.  See #41.

Reply:
See 0a3f4bc0587431f616a0d35548f9fc80da72a16b for changes to the way we are outputting. I implemented a logger with a handler for file output as well as stdout.

Reply:
Sorry I lost the thread here for a while, but thanks for taking care of that!